tmk_core/common/avr/suspend.c

```diff
@@ -167,6 +167,10 @@ void suspend_power_down(void) {
     rgb_matrix_set_suspend_state(true);
 #    endif
 
+#    if defined(LED_MATRIX_ENABLE)
```

Comment:
Let's swap these around to match everywhere else LED/RGB Matrix related stuff is conditionally added.